design/features/confirmation-email-for-form-fillers/README.md

```diff
@@ -0,0 +1,57 @@
+# Confirm that a form's been submitted (form fillers)
+
+## Status
+
+Date created: *2023-11-06*  
+
+* Epic trello card: https://trello.com/c/0svlNImf/1089-epic-confirm-that-a-forms-been-submitted  
+* Mural working board: https://app.mural.co/t/gaap0347/m/gaap0347/1688563053078/f59d07f0cd2dca526768595d30228b4b0188ea32?sender=ue1ef9fc3c2ad3697c3c93132 
+___
+
+## Contents
+
+- [Confirm that a form's been submitted (form fillers)](#confirm-that-a-forms-been-submitted-form-fillers)
+  - [Status](#status)
+  - [Contents](#contents)
+  - [What](#what)
+  - [Why](#why)
+  - [Hypothesis](#hypothesis)
+
+___
+
+<br>
+
+## What
+
+This feature introduces new functionality to the form filler's journey, allowing them to request an email confirming that they've successfully submitted a form. 
+
+### User stories
+
+**As a form creator**, I need to know how the content I write will be used, so that I know what information to include to help our form fillers.
+
+**As a form filler**, I need confirmation my form has been submitted, so that I feel confident that I have completed my task.
+
+**As a form filler**, I need a record of my submission, so that I have a reminder of when I sent my form, and know how to chase it up if I haven’t heard back.
```

Comment:
whats the difference between "form has been submitted" and "record of my submission"?

Reply:
One is a confirmation the form has been sent the other is a copy of the data they entered